### DISCUSSION TITLE
LtC Public Review Announcements

### DISCUSSION THREAD

**Discussion by ben-norton** [tdwg/ltc#48] [Announcements]
## 👋 Welcome!
Welcome to the Latimer Core Discussion Board, and thank you for participating in the Public Review!
The goal of this discussion is to help you get started, which begins with important logistical and scheduling information. 

## Announcements
_20240118_  
<a name="openforum-20240118"></a>
Final Open Forum
Thursday, January 18th, 2024
10am - 11am EST [Convert timezones](https://www.timeanddate.com/worldclock/converter.html?iso=20240118T150000&p1=207&p2=64&p3=270)
Zoom 
https://us02web.zoom.us/j/81814652210?pwd=SExxNzNiV2NWc1E3SzFnMXNmZzg5dz09
Meeting ID: 818 1465 2210
Passcode: 665891

_20240110_  
<a name="openforum-20240111"></a>
Open Forum 2
Thursday January 11th, 2024
10am - 11am EST [Convert timezones](https://www.timeanddate.com/worldclock/converter.html?iso=20240104T150000&p1=207&p2=64&p3=270)
Zoom 
https://us02web.zoom.us/j/81814652210?pwd=SExxNzNiV2NWc1E3SzFnMXNmZzg5dz09
Meeting ID: 818 1465 2210
Passcode: 665891

_20240103_  
<a name="openforum-20240104"></a>
Open Forum 1
Thursday January 4th, 2024
10am - 11am EST [Convert timezones](https://www.timeanddate.com/worldclock/converter.html?iso=20240104T150000&p1=207&p2=64&p3=270)
Zoom 
https://us02web.zoom.us/j/81814652210?pwd=SExxNzNiV2NWc1E3SzFnMXNmZzg5dz09
Meeting ID: 818 1465 2210
Passcode: 665891

Please download and import the following iCalendar (.ics) files to your calendar system.
Weekly: [https://us02web.zoom.us/meeting/tZUldu2uqzkoG9S_PO4WV5hWUfx3vspPz-uX/ics?icsToken=98tyKuGppjouHNGXshiHRpwcGoj4LO7wmHpBjfpuoy_KVxdlNwvBA-RaYaRsAOjK](https://us02web.zoom.us/meeting/tZUldu2uqzkoG9S_PO4WV5hWUfx3vspPz-uX/ics?icsToken=98tyKuGppjouHNGXshiHRpwcGoj4LO7wmHpBjfpuoy_KVxdlNwvBA-RaYaRsAOjK
)

_20231208_ 
1. The forum schedule with zoom meeting information is coming soon. See [Forums Schedule](#forums-schedule)
2. All comments posted to the Latimer Core discussion board during the week of December 24th (12/24 - 12/30) will receive a response the following week. 

## Public Review Procedures
We’re using Discussions as a place to ask questions about Latimer Core Standard.  If you have thoughts or questions about the Latimer Core (LtC) Standard during the public review, please submit your comment here:    
[Public Review Discussion Board](https://github.com/tdwg/ltc/discussions/categories/public-review)

### Class Discussions
The Public Review discussion is organized into categories based on the top-level classes in the Latimer Core standard. If your question or comment doesn't align with a specific class either in subject matter or scope, then please defer to the General category.

### Tags
When posting a comment or question, Please also tag your submission with the relevant [LtC class(es) or topics] (https://github.com/tdwg/ltc/issues/labels?sort=name-asc) (e.g. `Class:Address`).

## Open Forums Schedule Information<a id="forums-schedule" />
During the latter half of the public review period, the review manager will host weekly forums that are open to everyone. If you would prefer to ask questions or provide feedback directly to the Latimer Core team, then please join one of the open forums hosted on Zoom. As of Dec. 8th, the open forum schedule has yet to be finalized but will be before the public review officially begins. 

Thank you!

**Comment by jbstatgen** [comment]
Ratified we are!
